src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -153,6 +153,7 @@ public void Downvote()
                     (pair) =>
                         new Tuple<PackageHeader, PackageVersion>(
                         pair.Item1,
+                        //.First can throw an InvalidOperationException if the specified condition is not met.
                         pair.Item1.versions.First(x => x.version == pair.Item2)));
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.first?view=netframework-4.7.2
```
InvalidOperationException
No element satisfies the condition in predicate.
```

Reply:
Can happen if the dependencies do not exist anymore (filtered out due to ban or deprecated)

Reply:
deprecated dependencies are still available.  We should not be returning banned ones from the server.